Sources/MapboxMaps/Style/StyleURI.swift

```diff
@@ -29,21 +29,21 @@ public struct StyleURI: Hashable, RawRepresentable {
     }
 
     /// Mapbox Streets is a general-purpose style with detailed road and transit networks.
-    public static let streets = StyleURI(rawValue: "mapbox://styles/mapbox/streets-v11")!
+    public static let streets = StyleURI(rawValue: "mapbox://styles/mapbox/streets-v12")!
```

Comment:
It seems that globe is enabled by default in new styles - is this expected?

Reply:
yes its default behaviour for v12 styles.